dqn_training.py

```diff
@@ -21,20 +21,20 @@
 if len(sys.argv) > 2:
     agent1_file = 'agents/' + sys.argv[2]
 else:
-    agent1_file = 'agents/random_actions'
+    agent1_file = 'agents/State_Machine/random_actions_delay'
```

Comment:
Changed to train against the random_actions_delay agent (should provide more accurate win rate analysis against the Epsilon value curve).